twitter_timeline/main.py

```diff
@@ -23,22 +29,97 @@ def before_request():
 
 
 @app.route('/friendship', methods=['POST', 'DELETE'])
-@json_only
+#@json_only
```

Comment:
If you take this one out, and someone sends a not-json request, your code is gonna break